### DISCUSSION TITLE
Does the number of peaks in the accessbility matrix affect the results of pycisTopic?

### DISCUSSION THREAD

**Discussion by massonix** [aertslab/pycisTopic#183] [Q&A]
Hi pycisTopic team!

I have scATAC-seq data for multiple cancer patients. I'm analyzing them separately given the large interpatient variability. I have a quick question: does the number of peaks affect the results of pycisTopic? Each sample is relatively homogeneous (a "blob" in the umap), so I'm not particularly concerned of cell-type specific peaks. I have > 100K peaks, which makes the matrix really sparse.

Thanks in advance for your response!

**Comment by SeppeDeWinter** [comment]
Hi @massonix

Yes the number of peaks has an influence on the results but I'm not sure wether in your case this should be a concern or not.

Best,

S
